go.mod

```diff
@@ -1,5 +1,5 @@
 module github.com/fortanix/sdkms-client-go
 
-require (
-	github.com/pkg/errors v0.8.1
-)
+go 1.19
```

Comment:
Should this be the minimum Go version we support, i.e. 1.18?

Reply:
changed to 1.18